eng/pipelines/templates/stages/archetype-sdk-publish-js.yml

```diff
@@ -27,53 +27,47 @@ extends:
           dependsOn: ${{ parameters.BuildStageName }}
 
           jobs:
-            - deployment: Publish
-              environment: 'package-publish'
+            - job: PrePublish
+              variables:
+                - template: /eng/pipelines/templates/variables/image.yml
               pool:
-                name: azsdk-pool-mms-ubuntu-2004-general
-                image: azsdk-pool-mms-ubuntu-2004-1espt
+                name: $(LINUXPOOL)
+                image: $(LINUXVMIMAGE)
                 os: linux
+              steps:
+                - checkout: self
 
-              strategy:
-                runOnce:
-                  deploy:
-                    steps:
-                    - checkout: self
+                - download: current
+                  artifact: ${{ parameters.ArtifactName }}
+                  timeoutInMinutes: 5
 
-                    - download: current
-                      artifact: ${{ parameters.ArtifactName }}
-                      timeoutInMinutes: 5
+                - task: PowerShell@2
```

Comment:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4829962&view=logs&j=8764118a-3c4f-5698-0297-488ba3c20524&t=636bd5ed-7514-53fc-22c1-f02faa376752

Can we add a name to this step and also add some logging so we know what it is doing?